payments/src/main/java/nl/tudelft/wdm/group1/payments/events/OrderConsumer.java

```diff
@@ -0,0 +1,30 @@
+package nl.tudelft.wdm.group1.payments.events;
+
+import nl.tudelft.wdm.group1.common.Payment;
+import nl.tudelft.wdm.group1.common.Order;
+import nl.tudelft.wdm.group1.payments.PaymentRepository;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Service;
+
+@Service
+public class OrderConsumer {
+    private final PaymentRepository paymentRepository;
+    private final Producer producer;
+
+    private final Logger logger = LoggerFactory.getLogger(OrderConsumer.class);
+
+    public OrderConsumer(PaymentRepository paymentRepository, Producer producer) {
+        this.paymentRepository = paymentRepository;
+        this.producer = producer;
+    }
+
+    @KafkaListener(topics = {"orderCheckout"})
```

Comment:
This is slightly out of scope, but since we're creating new events anyway, it might be best to fix this now: Gerben made classes in `common` holding all topic strings. He made them for all microservices except for Payment (check `OrdersTopic` for an example). Could you perhaps duplicate one of the existing topic classes and adapt it to payments, and then use the created variables here?